### PR TITLE
website/integrations: Add Synology DSM

### PR DESCRIPTION
**Pull request by bobvmierlo**
## Details

Add Synology DSM as a client, using OpenID

---

## Checklist
-   [X] The documentation has been updated
-   [x] The documentation has been formatted

### CODE REVIEW

**Comment by tanberry** [comment]
Hi @bobvmierlo thanks so much for adding Synology's DSM to the list of integrations. I'll give it a quick copy-edit, and then after you made any suggested changes can you also please be sure to run the `make website` command, which runs our linter and will clear up the build failure for "lint-prettier" on current PR?

File: website/integrations/services/synology-dsm/index.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+To configure Synology DSM to utilize Authelia as an OpenID Connect 1.0 Provider:
+
+1. Go to DSM.
+2. Go to Control Panel.
+3. Go To Domain/LDAP.
+4. Go to SSO Client.
+5. Check the Enable OpenID Connect SSO service checkbox in the OpenID Connect SSO Service section.
+6. Configure the following values:
+7. Profile: OIDC
+8. Name: Authentik
```
[the remark this answers is not in the document]
Comment (tanberry) [reply website/integrations/services/synology-dsm/index.md:49]: I'd suggest that you make all of these options into bullet points...

**Review by tanberry — CHANGES_REQUESTED** [review]
Just formatting suggestions, and the request to run `make website` before pushing these or other changes. Thanks again for this contribution!

**Comment by tanberry** [comment]
Hi @bobvmierlo I forgot to mention, you will also need to edit the `sidebarsIntegrations.js` file, in order for the new Integration to appear in the ToC on left side. Let us know if you have any questions at all, and thanks again for this contribution!

**Comment by bobvmierlo** [comment]
Added the sidebar changes and ran make website command

**Comment by tanberry** [comment]
Thanks @bobvmierlo for running the linter; that part of build now passes.

Just 2 more clean-up tasks, then we can merge:

1. I think you only need two files in the PR, the new document for how to add Synology and the sidebar one. What are the other 58 files? Can you remove those from the PR, please?
2. For lines 51-56, you still need to make those into bullet items... I only did the first two for you. ;-)

**Comment by bobvmierlo** [comment]
> Thanks @bobvmierlo for running the linter; that part of build now passes.
> 
> Just 2 more clean-up tasks, then we can merge:
> 
> 1. I think you only need two files in the PR, the new document for how to add Synology and the sidebar one. What are the other 58 files? Can you remove those from the PR, please?
> 2. For lines 51-56, you still need to make those into bullet items... I only did the first two for you. ;-)

Tried to remove the unneeded files, they where a accidental addition due to running the make website command. Not sure if it worked though, might need some help with that. I'm not a git expert.

**Comment by tanberry** [comment]
Yeah, git can be a bit wild at times, haha. I have an entire repo just to hold my cheat-sheets for git. 

Looks like you figured out how to remove the extra files, though. 

It's rebuilding now, then I'll merge. Thanks again for your contribution and your perseverance!

**Review by tanberry — APPROVED** [review]
I'll make the alphabetical change later.

**Pull request merged**